ansible/kubernetes/update.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: system updates
+  hosts: k8s
+  become: true
+  tasks:
+    - name: print host
+      become: false
+      debug:
+        var: inventory_hostname
+
+    - name: drain hosts
+      become: false
+      environment:
+        KUBECONFIG: "{{ lookup('env', 'KUBECONFIG') }}"
+      kubernetes.core.k8s_drain:
+        name: "{{ inventory_hostname }}"
+        delete_options:
+          delete_emptydir_data: true
+          ignore_daemonsets: true
+          terminate_grace_period: 0
+      delegate_to: 127.0.0.1
+
+    - name: update packages
+      ansible.builtin.shell: yum update -y --disablerepo=kubernetes
```

Comment:
This will have to be updated when we migrate to Ubuntu.

Reply:
Yeah, I knocked this together on short notice on a maintenance day and couldn't figure out how to do everything we needed in a cross-distro manner.

Reply:
Without added more plays and vars, I should add. Branching based on OS isn't terribly difficult, just annoying when you're in a hurry.